Enterprise-CMCS/macpro-mdct-mfp
# MDCT-3018: RE&T Page Creation Pt. 1

### Description

This PR covers the initial creation of the `Recruitment, Enrollment, and Transitions` sub-sections for SAR, focusing mainly the first sub-section and the logic of generating number fields dynamically based on any "Other" target populations added in the Work Plan. 

Part of what this PR **does** cover is the sidebar sections, navigation/routing, and the copy for the headings and subheadings per sub-section.

`Follow-up items (not covered in this PR):`
- Logic for rendering number fields within multiple quarters
- Logic for rendering the correct reporting periods

### Related ticket(s)


CMDCT-3018

---
### How to test

- Log in as a state user
- Create a new Work Plan
- Add an "other" target population
- Create a SAR
- Navigate to `/sar/recruitment-enrollment-transitions/*`

The top-level copy should match Figma and you should see a number / text field for each additional target population.

### Important updates


N/A

---
### Author checklist


---


_convert to a different template: [test → val](?expand=1&template=test-to-val-deployment.md)_ | _[val → prod](?expand=1&template=val-to-prod-deployment.md)_
